package.json

```diff
@@ -111,7 +111,7 @@
     "normalize.cssinjs": "1.1.0",
     "polished": "3.4.0",
     "react-svg": "7.2.2",
-    "suomifi-design-tokens": "0.6.0",
+    "suomifi-design-tokens": "0.7.1",
```

Comment:
Why go from 0.6.0 to 0.7.1? 0.7.0 should be sufficient?

Reply:
0.7.1 had one more name change that was due to erroneous documentation for 0.7.0.

Reply:
Ah, this branch is not rebased with the current develop, so it shows the diff wrong (current develop is 0.7.0, not 0.6.1). Might be worth rebasing even though there are no conflicts.

This should then be 0.8.0 because of breaking changes.

Reply:
And the version raise should actually be with a separate PR...

Reply:
Might be problematic to update version with the changes PR as then it depends in which order they are then able to merged / or other PR's version need to be adjusted if something is merged before it etc.

I would suggest to continue to increase version only on separate PR.

Reply:
Are you looking at the right version number? That 0.7.1 is the version of the tokens. I did not touch the ui-components version number which is still 0.7.0.

And yes, it is rebased :)

Reply:
The next ui-components release should be 0.8.0.

Reply:
😂 My bad! Was indeed looking at wrong version number. So no issues with dependencies, a separate PR is required anyways to increase the minor version for ui-comps.